tests/modules/selenium/test_selenium_builder.py

```diff
@@ -658,3 +666,407 @@ def test_resize_window(self):
 
         for idx in range(len(target_lines)):
             self.assertIn(target_lines[idx], content, msg="\n\n%s. %s" % (idx, target_lines[idx]))
+
+    def test_mix_syntax(self):
+        self.configure({
+            "execution": [{
+                "executor": "apiritif",
+                "scenario": "loc_sc"}],
+            "scenarios": {
+                "loc_sc": {
+                    "requests": [{
+                        "label": "la-la",
+                        "actions": [
+                            {
+                                "type": "click",
+                                "locators": [
+                                    {"name": "btn1"},
+                                ]
+                            },
+                            {"typeById(Id_123)": "London"}
+                        ]}]}}})
+
+        self.obj.prepare()
+        with open(self.obj.script) as fds:
+            content = fds.read()
+
+        target_lines = [
+            "var_loc_keys=self.loc_mng.get_locator([{'name':'btn1',}])self.driver.find_element"
+            "(var_loc_keys[0],var_loc_keys[1]).click()",
+            "var_loc_keys=self.loc_mng.get_locator([{'id':'Id_123',}])self.driver.find_element"
+            "(var_loc_keys[0],var_loc_keys[1]).clear()",
+            "self.driver.find_element(var_loc_keys[0],var_loc_keys[1]).send_keys('London')"
+        ]
+
+        for idx in range(len(target_lines)):
+            self.assertIn(TestSeleniumScriptGeneration.clear_spaces(target_lines[idx]),
+                          TestSeleniumScriptGeneration.clear_spaces(content),
+                          msg="\n\n%s. %s" % (idx, target_lines[idx]))
+
+    def test_syntax2_all_multiple_locators_case_insensitive(self):
```

Comment:
As I see this test doesn't really check if locators are case insensitive (cos real conversion will be made in run-time). So could you fix it or remove?

Reply:
That is actually a good catch. I will remove this test for now.